### PR TITLE
Update rubocop and refactor the code accordingly

### PR DESCRIPTION
**Pull request by nemodjuric**
Connects to #158 
I mimicked .rubocop.yml from caseflow-certification repo with few exceptions as ruby version and few directories that don't exist in feedback.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
   Exclude:
     - 'db/**/*'
     - 'config/**/*'
-    - 'script/**/*'
     - 'vendor/**/*'
 
 Style/StringLiterals:
   EnforcedStyle: double_quotes
 
+Style/FrozenStringLiteralComment:
```
Comment (dannysellers) [inline .rubocop.yml]: Seems like this should be enabled / was enabled during the lint. There's an `always` option, or `when_needed` is the default ([ref](https://rubocop.readthedocs.io/en/latest/cops_style/#stylefrozenstringliteralcomment)).

**Review by dannysellers — CHANGES_REQUESTED** [review]
lgtm with one question

File: .rubocop.yml (hunk 1 above)
Comment (nemodjuric) [reply .rubocop.yml]: I might have run 'rake lint' before updating .rubocop.yml 
I'll open another PR for this.

Comment (dannysellers) [reply .rubocop.yml]: Oh, I meant—is there any reason we shouldn't enable this cop? Freezing strings apparently [improves performance slightly](https://bugs.ruby-lang.org/issues/8976#note-30) and will be the [default behavior in Ruby 3](https://bugs.ruby-lang.org/issues/11473).

**Pull request closed without merging**